content/en/docs/concepts/overview/working-with-objects/annotations.md

```diff
@@ -69,6 +69,23 @@ If the prefix is omitted, the annotation Key is presumed to be private to the us
 
 The `kubernetes.io/` and `k8s.io/` prefixes are reserved for Kubernetes core components.
 
+For example, the sample Pod with the annotation `imageregistry: https://hub.docker.com/`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cuda-test
+  annotations:
+    imageregistry: "https://hub.docker.com/"
+spec:
+  containers:
+    - name: cuda-test
+      image: "k8s.gcr.io/cuda-vector-add:v0.1"
```

Comment:
The image used doesn't look like coming from docker hub, so it doesn't look like a good example to me.

Reply:
Added more generic example which takes image from docker hub.